.evergreen/config.yml

```diff
@@ -1625,6 +1671,46 @@ tasks:
           AUTH: "noauth"
           SSL: "nossl"
 
+  - name: "test-kms-tls-invalid-cert"
+    tags: ["kms-tls"]
+    commands:
+      - func: bootstrap-mongo-orchestration
+        vars:
+          TOPOLOGY: "server"
+          AUTH: "noauth"
+          SSL: "nossl"
+      - func: start-kms-mock-server
+        vars:
+          KMS_CA_FILE: "ca.pem"
+          KMS_CERT_FILE: "expired.pem"
+      - func: run-kms-tls-test
+        vars:
+          KMS_TLS_INVALID_CERT: "true"
+          KMS_TLS_INVALID_HOSTNAME: "false"
```

Comment:
I'm using these two environment variables to control which test (invalid cert, or invalid hostname) should run. I could instead launch two mock kms servers with different ca and cert files and only use a single evg task if that would be better.

Reply:
I'm not too concerned with having separate tasks per test case since there are only two. If and when we add more test cases, we could consolidate the test cases and launch multiple mock KMS servers on different ports or figure out a way to cleanly restart the mock KMS server after each test case.

However, since these environment variables are mutually exclusive, can we consolidate it into one environment variable, like:
```
KMS_TLS_TESTCASE="INVALID_CERT"
```

Reply:
That seems fair to me! Consolidated to a single `KMS_TLS_TESTCASE` variable.